### ISSUE TITLE
[Feature-Request] Add ssl option also along with the deployment

### ISSUE THREAD

**Issue by appunni-dishq** [nezhar/wordpress-docker-compose#48]

**Comment by nezhar** [comment]
Do you consider this an option for a production, staging or development environment?
Can this be used to generate an SSL certificate when running locally?

**Comment by appunni-dishq** [comment]
No. It cannot be used in local. But it will work if you add localhost as exception in browser

**Comment by cagnulein** [comment]
tested and it works flawlessy :)

**Comment by benjaminloeffel** [comment]
Adding SSL support for local development environment would be a great addition to this repository!

**Comment by seyfer** [comment]
I think the use of letsencrypt and the certbot console tool will be better. There is an example
https://www.digitalocean.com/community/tutorials/how-to-install-wordpress-with-docker-compose

**Comment by nezhar** [comment]
We may add a not in the readme how such a setup may look with nginx. Here is an example from another project: https://github.com/snypy/snypy-docker?tab=readme-ov-file#make-the-containers-publicly-avaialble
